NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 24.0.4 2022-08-11
+
+* Fixing mb bugfest and clean deployment schema difference version modified ([MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935))
```

Comment:
Given that this is documentation for an operator or user of the module, I think it is better to use language that states what has happened.

```suggestion
* Adds integrity checks for statistical code types during upgrade ([MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935))
```

Reply:
Description updated